src/components/services/Events.vue

```diff
@@ -1,39 +1,113 @@
+<script>
+  import CustomHeader from '@/components/shared/CustomHeader.vue';
+  export default {
+    components: {
+      CustomHeader,
+    },
+    data() {
+       return {
+          items: [
+            {message: 'La Scala Opera Trip'},
+            {message: 'Night Club Hop'}
```

Comment:
To account for all the info we want to show here, update these objects to be in this format: 
```
{
     title: 'La Scala Opera Trip',
     reservations: 26,
     openings: 14,
     transportation: 'ABC Limo',
     host: 'Andy Cohen'
}
```